examples/DB/Zookeeper/test_plan.py

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env python
+"""
+Demostrates Zookeeper driver usage from within the testcases.
+"""
+
+import os
+import sys
+
+try:
```

Comment:
As we discussed, let's remove this try-except block (maybe add internal dependencies first?)